.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,94 +1,71 @@
 name: 🐛 Bug
-description: Report a bug or incorrect behavior
-title: "[Bug] <title>"
+description: Report a bug or an incorrect behavior
 labels:
   - bug
-  - good first issue
-assignees:
-  - KRTirtho
+
 body:
   - type: checkboxes
     attributes:
       label: Is there an existing issue for this?
-      description: Please search to see if an issue already exists for the bug you encountered.
+      description: Make sure to check if this issue is a duplicate.
       options:
         - label: I have searched the existing issues
           required: true
   - type: textarea
     attributes:
       label: Current Behavior
-      description: Tell us what you're experiencing concisely.
+      description: Type what you are experiencing currently.
     validations:
       required: true
   - type: textarea
     attributes:
       label: Expected Behavior
-      description: Let us know what you expected to happen concisely.
+      description: Write what you expected to happen.
     validations:
       required: true
   - type: textarea
     attributes:
-      label: Steps To Reproduce
-      description: Steps to reproduce the behavior.
+      label: Steps to reproduce
+      description: Steps to reproduce the issue. A not well written description might delay the resolve of it.
       placeholder: |
-        1. In this environment...
-        2. With this config...
-        3. Run '...'
-        4. See error...
+        1. I opened the app
+        2. I did this
+        3. And that
+        4. Then this happened
     validations:
-      required: false
+      required: true
   - type: input
-    attributes:
-      label: Spotube Version
-      description: In which version of Spotube did you encounter this bug?
-      placeholder: e.g. v2.3.0
-  - type: dropdown
-    attributes:
+      attributes:
       label: Operating System
-      description: In which OS you're experiencing the bug?
```

Comment:
Same here. Why remove this? We don't want BSD users trying Spotube on BSD but failing and creating an issue here not knowing we don't support BSD :joy:

Reply:
Keep the options